src/server/scripts/Kalimdor/CavernsOfTime/TheBlackMorass/the_black_morass.cpp

```diff
@@ -64,9 +64,15 @@ enum medivhMisc
     EVENT_OUTRO_5               = 14,
     EVENT_OUTRO_6               = 15,
     EVENT_OUTRO_7               = 16,
-    EVENT_OUTRO_8               = 17
+    EVENT_OUTRO_8               = 17,
+
+    FACTION_KEEPERS_OF_TIME     = 989
```

Comment:
Is this still used? Or not needed anymore?
`FACTION_KEEPERS_OF_TIME     = 989`?